frontend/app/protected/template.hbs

```diff
@@ -1 +1 @@
-{{liquid-outlet}}
+{{outlet}}
```

Comment:
Hmm, I'm not sure if this is the correct fix. It seems that when using liquid-fire, your `{{outlet}}` tag should be replaced with `{{liquid-outlet}}` (see http://ember-animation.github.io/liquid-fire/#/cookbook). Maybe we just have one (liquid-) outlet too much that should be removed?